src/common/paths/query_ast.yml

```diff
@@ -13,20 +38,58 @@ post:
         enum:
           - application/json
   requestBody:
-    description: Analyzed Flux query to generate abstract syntax tree.
+    description: Analyzed Flux query to generate an AST.
     content:
       application/json:
         schema:
           $ref: "../schemas/LanguageRequest.yml"
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl --request POST 'INFLUX_URL/api/v2/query/ast' \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json \
+        --header 'Authorization: Token INFLUX_TOKEN' \
+        --data 'from(bucket: "example-bucket")
+                  |> range(start: -5m)
+                  |> filter(fn: (r) => r._measurement == "example-measurement")'
   responses:
     "200":
-      description: Abstract syntax tree of the flux query.
+      description: |
+        Successful request.
+        AST of the Flux query.
       content:
         application/json:
           schema:
             $ref: "../schemas/ASTResponse.yml"
+          examples:
+            $ref: "../schemas/ASTSuccessResponse.yml"
+    "400":
+      description: |
+        Bad request.
+        InfluxDB is unable to parse the request.
+        The response body contains detail about the problem.
+      headers:
+        X-Platform-Error-Code:
+          description: |
+            The reason for the error.
+          schema:
+            type: string
+            example: "invalid"
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/Error.yml"
+          examples:
+            invalidASTValue:
+              summary: Invalid AST
+              value: {
+                  "code": "invalid",
+                  "message": "invalid AST: loc 1:6-1:19: missing property key"
+                }
```

Comment:
Might be helpful to include in the summary where the key was missing, what property.